packages/ui/src/hooks/use-password-error-message.ts

```diff
@@ -0,0 +1,78 @@
+import { type PasswordRejectionCode, type PasswordIssue } from '@logto/core-kit';
+import { type RequestErrorBody } from '@logto/schemas';
+import { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import useListTranslation from '@/hooks/use-list-translation';
+
+/**
+ * Return an object with two functions for getting the error message from an array of {@link PasswordIssue} or a {@link RequestErrorBody}.
+ */
+const usePasswordErrorMessage = () => {
```

Comment:
Since it does not have any runtime requirements. Should we define the `getErrorMessage` and `getErrorMessageFromBody` as two static util methods instead of hooks?

Reply:
I'm not sure which is better, since this can avoid necessary function creation

Reply:
discussed offline, keep as it is for now